cognite/client/data_classes/extractionpipelines.py

```diff
@@ -30,27 +30,24 @@ class ExtractionPipelineContact(CogniteObject):
     """A contact for an extraction pipeline
 
     Args:
-        name (str): Name of contact
-        email (str): Email address of contact
-        role (str): Role of contact, such as Owner, Maintainer, etc.
-        send_notification (bool): Whether to send notifications to this contact or not
+        name (str | None): Name of contact
+        email (str | None): Email address of contact
+        role (str | None): Role of contact, such as Owner, Maintainer, etc.
+        send_notification (bool | None): Whether to send notifications to this contact or not
     """
 
-    def __init__(self, name: str, email: str, role: str, send_notification: bool) -> None:
+    def __init__(
+        self,
+        name: str | None = None,
+        email: str | None = None,
+        role: str | None = None,
+        send_notification: bool | None = None,
```

Comment:
Is this class used as both a read and write class? If only read, we don't really want the defaults.

Reply:
It is used for both.